src/OSPSuite.SimModel/src/SimulationOptions.cpp

```diff
@@ -15,13 +15,14 @@ using namespace std;
 
 SimulationOptions::SimulationOptions(void)
 {
-	_validateXMLWithSchema = true;
+	_validateXMLWithSchema = false; //perform schema validation only if explicitly set by user
```

Comment:
this has nothing to do with used parameters.
Just changed the default from true (="validate XML with SimModel-Schema") to false. Just to increase the performance.
PK-Sim/MoBi always set it to false (assuming models generated there are always valid :)
Matlab/R-Toolbox still can set it to true if required